addons/popochiu/engine/objects/character/popochiu_character.gd

```diff
@@ -683,9 +683,9 @@ func play_animation(animation_label: String, animation_fallback := 'idle'):
 		# Search for a valid animation corresponding to animation_label
 		_current_animation = _get_valid_oriented_animation(animation_label)
 		# If is not present, do the same for the the fallback animation.
-		if _current_animation == null: _current_animation = _get_valid_oriented_animation(animation_fallback)
+		if _current_animation == "null": _current_animation = _get_valid_oriented_animation(animation_fallback)
```

Comment:
Instead of asking for `"null"`, I think it will be better if we ask for `is_empty()`.

```suggestion
		if _current_animation.is_empty():
			_current_animation = _get_valid_oriented_animation(animation_fallback)
```